ufl/cell.py

```diff
@@ -84,6 +84,13 @@ def __lt__(self, other):
 # Mapping from cell name to topological dimension
 cellname2dim = dict((k, len(v) - 1) for k, v in num_cell_entities.items())
 
+# TODO: Remove this dict
+cellname2facetname = {"interval": "vertex",
```

Comment:
I believe that this is needed only in one file in FFC, you may want to add it there rather than adding it back here.
https://github.com/fem-on-colab/fem-on-colab/blob/main/fenics/patches/08-add-cellname2facetname-to-ffc